### PR TITLE
Download must-gather diagnostics - server side

### PR DESCRIPTION
**Pull request by pkliczewski**
Currently, in the case of an issue, the customer needs to figure out on their own where they need to go to get the diagnostics data of each applicable component. It would be a major User Experience
improvement to have this automated and fully integrated into the UI. This enhacement indicate how server side should be implemented.

### CODE REVIEW

**Comment by deads2k** [comment]
please update for my comments on the google doc.  We like adding commits instead of squashing during these reviews.

File: enhancements/download_report.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+title: Download must-gather diagnostics - server side
+authors:
+  - "@masayag"
+  - "@nmagnezi"
+  - "@pkliczewski"
+reviewers:
+  - "@deads2k"
+  - "@derekwaynecarr"
+  - "@soltysh"
+  - "@mfojtik"
```
Comment (deads2k) [inline enhancements/download_report.md]: Add @sanchezl as a reviewer please.

**Comment by deads2k** [comment]
@sferich888 should be tagged  for awareness
@spadgett will need to be aware if you want the openshift webconsole to make changes.

**Comment by pkliczewski** [comment]
> @spadgett will need to be aware if you want the openshift webconsole to make changes.

I think @jelkosz's team will implement changes in the UI

File: enhancements/download_report.md (hunk 1 above)
Comment (pkliczewski) [reply enhancements/download_report.md]: Added

**Comment by smarterclayton** [comment]
At the current time I don't think we're ready to do this from the console and given the RHHI focus change I think this should be deferred from 4.3 and picked up after we have stabilized more aspects of must-gather.

**Comment by pkliczewski** [comment]
@smarterclayton As far as I understand  @jelkosz's team is assigned to implement the change. Let's see whether they still have this change scheduled for 4.3. My understanding is that RHHI focus changes do not affect this work.

**Comment by russellb** [comment]
@pkliczewski I found a design review here: https://github.com/openshift/openshift-origin-design/pull/278 but I can't find anything where the implementation was done (or even tracked).  Did you see any more work happen on this?  or should we be closing this until it's potentially picked up again in the future?

**Comment by pkliczewski** [comment]
@russellb There was no implementation. It was dropped. We can close

**Pull request closed without merging**